src/Microsoft.AspNet.Razor/CodeGenerators/CSharpTagHelperCodeRenderer.cs

```diff
@@ -659,7 +659,9 @@ private void BuildBufferedWritingScope(Chunk htmlAttributeChunk, bool htmlEncode
                 // Scopes are a runtime feature.
                 if (!_designTimeMode)
                 {
-                    _writer.WriteMethodInvocation(_tagHelperContext.StartTagHelperWritingScopeMethodName);
+                    _writer.WriteMethodInvocation(
+                        _tagHelperContext.StartTagHelperWritingScopeMethodName,
+                        "encoder: null");
```

Comment:
part of me would rather an overload without one. But I guess that's overkill for just this. Ya don't do named params here.